Description: Document undocumented options
 lsar's -L and -v options and unar's -v option are not documented in
 their respective manual pages, so do so.
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/unar/+bug/1393321
Bug: https://code.google.com/p/theunarchiver/issues/detail?id=352
Author: Matt Kraai <kraai@ftbfs.org>
Last-Update: 2014-12-15

Index: unar/Extra/lsar.1
===================================================================
--- unar.orig/Extra/lsar.1	2014-12-15 07:00:54.285823351 -0800
+++ unar/Extra/lsar.1	2014-12-15 07:01:24.095917401 -0800
@@ -11,6 +11,9 @@
 .BR \-l ", " \-long
 Print more information about each file in the archive.
 .TP
+.BR \-L ", " \-verylong
+Print all available information about each file in the archive.
+.TP
 .BR \-t ", " \-test
 Test the integrity of the files in the archive, if possible.
 .TP
@@ -49,5 +52,8 @@
 .TP
 .BR \-h ", " \-help
 Display help information.
+.TP
+.BR \-v ", " \-version
+Print version and exit.
 .SH SEE ALSO
 .BR unar (1)
Index: unar/Extra/unar.1
===================================================================
--- unar.orig/Extra/unar.1	2014-12-15 07:00:54.285823351 -0800
+++ unar/Extra/unar.1	2014-12-15 07:01:24.095917401 -0800
@@ -68,6 +68,9 @@
 .BR \-q ", " \-quiet
 Run in quiet mode.
 .TP
+.BR \-v ", " \-version
+Print version and exit.
+.TP
 .BR \-h ", " \-help
 Display help information.
 .SH SEE ALSO
